src/Nancy.Demo.Authentication.Forms.TestingDemo/LoginFixture.cs

```diff
@@ -17,10 +17,10 @@ public LoginFixture()
         }
 
         [Fact]
-        public void Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
+        public async Task Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
         {
             // Given, When
-            var response = browser.Post("/login/", (with) =>
+            var response = await browser.Post("/login/", (with) =>

```

Comment:
Waiting is exactly what the existing code does/did though so I don't think it will be an issue - obviously it will be marked as deprecated, but I think if *everyone's* tests break with a 2.0 upgrade, and it means editing every test to fix, it will piss off and put off an awful lot of our users.

I would rather that shim than postasync personally - @thecodejunkie ?